students/mark_mcduffie/lesson2/GridPrinter.py

```diff
@@ -0,0 +1,52 @@
+#Grid Printer part 2
+#prints grid of any defined size
+def print_grid(size):
+    x = (size-1)/2
+    x = int(x)
+    y = (size-1)
+    y = int(y)
+    for i in range(y):
+        if(i == 0 or i == (x) or i == y):
+            plusLine(size)
+            middleLine(size)
+        else:
+          middleLine(size)
+    plusLine(size)
+def plusLine(size):
```

Comment:
make sure to use snake_case for method names and variable names

```suggestion
def plus_line(size):
```